src/utils/sentry/index.ts

```diff
@@ -15,5 +15,5 @@ export const sentryTracker = new BrowserClient({
   // We recommend adjusting this value in production
   tracesSampleRate: 1.0,
   // do not edit this, it is subject to be changed automatically by scripts/release.js
-  release: '<VERSION>',
+  release: 'v1-sdk@1.10.0-d28fd6e',
```

Comment:
revert this. You are probably running builds locally

Reply:
https://github.com/Voltz-Protocol/v1-sdk/pull/132/commits/e7f9f39f084d544772329f7821ec4878cfd5c4fc